#3300: triu, tril and #3276: update empty and #2583 pool 2d enable

muthutt opened the pull request:
#3300: triu, tril and 
#3276: update empty and #2583 pool 2d enable 
#0: enable more tests to run on WHB0

muthutt committed:
#2583: pool 2d fails in WH B0
     - overall WH B0 uplift enables this test
   #0: remove skip_wormhole bits
   #0: fix wormhole b0 filtering to run more tests on WHB0

Commit:
#3300: TT Metal operator support for triu, tril
- use a trick to get tril, triu to work out ok
- tril, triu implementation
- add triu, tril operations

Commit:
#3276: update empty to bind to C++ empty and bind

tt-aho reviewed (changes requested):
mostly lgtm. Minor comments but they should be addressed before merge

muthutt committed:
#0: update for bug fixes and reviews
#0: update for testing
#0: update empty function doc
#0: update doc for triu, tril
